README.md

```diff
@@ -388,14 +384,12 @@ limitations under the License.
   * The managability team at Juniper Networks.
 
 
-[img-pyangbind]: http://rob.sh/img/pyblogo_gh.png
```

Comment:
This image is still available at `https://cdn.rob.sh/img/pyblogo_gh.png` [(link)](https://cdn.rob.sh/img/pyblogo_gh.png).

Reply:
Done